salt/loader/lazy.py

```diff
@@ -1,9 +1,4 @@
-"""
-The Salt loader is the core to Salt's plugin system, the loader scans
-directories for python loadable code and organizes the code into the
-plugin interfaces used by Salt.
-"""
-
+import contextvars
```

Comment:
Why not
```python
try:
    # Try the stdlib C extension first
    import _contextvars as contextvars
except ImportError:
    # Py<3.7
    import contextvars
```
?

Reply:
For this import it doesn't matter since both stdlib and the backport have the same API we are using in this module. It only came up for salt ssh because of our thin dir processing.

Reply:
Ok.